src/MagicLogin.php

```diff
@@ -243,7 +248,12 @@ public function handleMagicLoginAfterUserSave(ActionEvent $event)
             return;
         }
 
-        // TODO: Ensure we are not creating in admin area.
+        // If user is logged in already then don't proceed.
```

Comment:
If user that is logged in is the user that has just been saved then we should still proceed as the configuration option for logging a user in as part of registration is switched on.